modules/ROOT/pages/sso-use-case-1.adoc

```diff
@@ -1,12 +1,14 @@
-= Configuring SSO When No Identity Provider Is Configured in Anypoint
+= Configuring SSO When No Identity Provider Is Configured in Anypoint Platform
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-This use case includes all of the instructions for setting up an identity provider for your portal when no other identity provider is configured in Anypoint Platform. You want to configure an identity provider as a login option for your portal. Since there is no identity provider configured in Anypoint Platform, the default identity provider automatically maps the identity provider for the portal to the identity provider in Anypoint Platform. This enables every Salesforce user to have an identity in Anypoint Platform.
+This use case includes instructions for setting up an identity provider for your portal when no other identity provider is configured in Anypoint Platform. You want to configure an identity provider as a login option for your portal. Since there is no identity provider configured in Anypoint Platform, the default identity provider automatically maps the identity provider for the portal to the identity provider in Anypoint Platform. This enables every Salesforce user to have an identity in Anypoint Platform. See xref:enabling-sso-for-your-portal.adoc#out-of-the-box-configurations[Out-of-the-Box Configurations] for more information about the default identity provider.
```

Comment:
```suggestion
If you haven't configured another identity provider in Anypoint Platform, you can use the default identity provider. The default provider automatically maps the identity provider for the portal to the identity provider in Anypoint Platform, providing every Salesforce user an identity in Anypoint Platform. See xref:enabling-sso-for-your-portal.adoc#out-of-the-box-configurations[Out-of-the-Box Configurations] for more information about the default identity provider.
```

Reply:
Saying, "...you can use the default identity provider" may imply that you can choose to use it. This is not the case here. This happens automatically if aeh_portal_idp is not set in the identity provider. Since there is no identity provided set up already, AEH cannot find aeh_portal_idp so it automatically uses the default idp.